app/models/like.rb

```diff
@@ -1,2 +1,9 @@
 class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
+    post.likes_counter = post.likes.count
+    post.save
+  end
 end
```

Comment:
- [ ]  Similar to the previous model, You have added the `update_likes_counter` method but it can be improved using the `after_save` callback and the increment method. :blush: 

  Suggested Changes:

```ruby
class Like < ApplicationRecord
  belongs_to :user
  belongs_to :post

  after_save :update_likes_counter

  private

  def update_likes_counter
    post.increment!(:likes_counter)
  end
end
```